Pruebas unitarias y validators - Feature campañas

LuisJuarez17 opened the pull request:
Creación de las pruebas unitarias respectivas y validators para campañas. Cambios en las rutas y controladores.

LuisJuarez17 committed:
Creación de validadores, test unitrarios.

DJPORTILLO30 commented on test/helper/helperData.js:
```diff
@@ -0,0 +1,14 @@
+
+  const testDataCampaign = {
```
la información no coincide con el modelo.

DJPORTILLO30 reviewed (changes requested):
Corregir pruebas unitarias ya que no estan siendo aprobadas.

LuisJuarez17 committed:
Corrección de errores de test

Se modificó la ruta utilizada para las pruebas unitarias y los datos enviados en las mismas. Se eliminó un validator que no se utilizaba.

Co-Authored-By: larochem20 <70065176+larochem20@users.noreply.github.com>
Co-Authored-By: Endercarias <50668460+Endercarias@users.noreply.github.com>

HJOC15 commented on models/campañas/campañas.js:
```diff
@@ -3,36 +3,40 @@ const { DataTypes } = require("sequelize");
     },
-    phone: {
-        type: DataTypes.NUMBER,
+    nombreCampania: {
```
Decidir si será campaña o campania, no pueden estar las dos.

HJOC15 replied:
Cambiar nombres de español a inglés.

HJOC15 reviewed (changes requested):
Revisar lo comentado.

LuisJuarez17 committed:
Creación de query

Query para la creación de la tabla de campañas

Co-Authored-By: Endercarias <50668460+Endercarias@users.noreply.github.com>
Co-Authored-By: larochem20 <70065176+larochem20@users.noreply.github.com>

DJPORTILLO30 committed:
Merge branch 'test-branch' into feature-Campañas

DJPORTILLO30 reviewed (approved):
Todo okay

HJOC15 reviewed (approved):
¡Buenas pruebas unitarias, cracks!

SFMZ21 reviewed (approved):
Todo ok
